src/components/pages/Offers/Offer/Stocks/ActivationCodesUploadDialog/ActivationCodesCsvForm/ActivationCodesCsvForm.jsx

```diff
@@ -41,6 +43,31 @@ export const ActivationCodeCsvForm = forwardRef(function ActivationCodeCsvForm(
         </div>
       </div>
       <div className="activation-codes-upload-separator" />
+      <div className="activation-codes-upload-template-section">
+        <p className="activation-codes-upload-gabarit">
+          {'Gabarits'}
+        </p>
+        <p>
+          <a
+            className="quaternary-link"
```

Comment:
On devrait ajouter ici un title explicite qui indique que l'on va télécharger un fichier.
On peux ajouter aussi 
```suggestion
            className="quaternary-link"
            type="text/csv"
```

Reply:
Done